pybliometrics/scopus/abstract_citations.py

```diff
@@ -191,7 +197,7 @@ def __str__(self):
         if len(authors) > 1:
             authors[-1] = " and ".join([authors[-2], authors[-1]])
         s = f"Document '{self.title}' by {', '.join(authors)}\npublished in "\
-            f"'{self.publicationName}' has the following citation trajectory "\
+            f"'{self.publicationName}' has the following {self._citation} citation trajectory "\
```

Comment:
How would this look? "has the following exclude-self citation trajectory"? Doesn't look readable. What about a dictionary approach, mapping the two accepted values to labels used in the __str__? One could combine this with a check whether the provided values are among those the Scopus API accepts

Reply:
I think "publication has the following citation trajectory without self-citations" or "publication has the following citation trajectory without books" should look more readable, changed that in the last commit.